src/main/java/com/hubspot/maven/plugins/dependency/management/DependencyManagementAnalyzer.java

```diff
@@ -186,6 +185,7 @@ private RequireManagmentConfig getEffectiveRequireManagementConfig(String key) {
   }
 
   private void logViolation(String message) {
+    violationLogger.accept("------------------------------------------------------------------------");
```

Comment:
I think that this method should still just do `violationLogger.accept` and you can add another helper called something like `logViolationMessage` or something that adds the hyphens.

Reply:
On second thought, it's probably easiest to just add a `logViolation(""------------------------------------------------------------------------")` the the `forEach`.